x-pack/plugins/maps/public/classes/sources/mvt_single_layer_vector_source/mvt_single_layer_vector_source.tsx

```diff
@@ -223,6 +227,10 @@ export class MVTSingleLayerVectorSource
     }
     return tooltips;
   }
+
+  async loadIsEditable(): Promise<boolean> {
```

Comment:
loadIsEditable implementation not needed on MVTSingleLayerVectorSource since it extends AbstractSource and the implementation is the same as AbstractSource

Reply:
Same comment as above. We need it since we're ultimately implementing `IVectorSource`